lib/labels.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+
+const https = require('https');
+const querystring = require('querystring');
+
+const request = Symbol('request');
+const getPath = Symbol('getPath');
+
+const parseLink = link => {
```

Comment:
To make it more descriptive:
```suggestion
const parseLinkHeader = link => {
```